tests/Mockery/Adapter/Phpunit/TestListenerTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Mockery
+ *
+ * LICENSE
+ *
+ * This source file is subject to the new BSD license that is bundled
+ * with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://github.com/padraic/mockery/blob/master/LICENSE
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to padraic@php.net so we can send you a copy immediately.
+ *
+ * @category   Mockery
+ * @package    Mockery
+ * @subpackage UnitTests
+ * @copyright  Copyright (c) 2010 Pádraic Brady (http://blog.astrumfutura.com)
+ * @license    http://github.com/padraic/mockery/blob/master/LICENSE New BSD License
+ */
+
+class Mockery_Adapter_Phpunit_TestListenerTest extends PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        // We intentionally test the static container here. That is what the
+        // listener will check.
+        $this->container = \Mockery::getContainer();
+        $this->listener = new \Mockery\Adapter\Phpunit\TestListener();
+        $this->testResult = new \PHPUnit_Framework_TestResult();
+        $this->test = new \Mockery_Adapter_Phpunit_EmptyTestCase();
+
+        $this->test->setTestResultObject($this->testResult);
+        $this->testResult->addListener($this->listener);
+
+        $this->assertTrue($this->testResult->wasSuccessful(), 'sanity check: empty test results should be considered successful');
+    }
+
+    public function testSuccessOnClose()
+    {
+        $mock = $this->container->mock('foo');
```

Comment:
Using `'foo'` here is causing some flaky tests to when running the full suite, just calling `mock()` for these purposes should be fine. Same on lines 56 and 71.